eduiddashboard/views/nins.py

```diff
@@ -523,13 +523,12 @@ def get_template_context(self):
 
         settings = self.request.registry.settings
         enable_mm = settings.get('enable_mm_verification')
-        session_user = get_session_user(self.request, raise_on_not_logged_in = False)
 
         context.update({
-            'nins': session_user.nins.to_list_of_dicts(),
-            'not_verified_nins': get_not_verified_nins_list(self.request, session_user),
+            'nins': self.user.nins.to_list_of_dicts(),
+            'not_verified_nins': get_not_verified_nins_list(self.request, self.user),
             'active_nin': self.get_active_nin(),
-            'has_mobile': has_confirmed_mobile(session_user),
+            'has_mobile': has_confirmed_mobile(self.user),
```

Comment:
I think that at least in the FormView's it should be safe to use self.user, that has just been fetched when constructing the view class. Following the code, everything seems correct, and the chance of of getting stale (not yet updated) data from the db to fill the templates seems the higher risk.